google/cloud/forseti/common/gcp_api/bigquery.py

```diff
@@ -237,6 +238,7 @@ def get_dataset_access(self, project_id, dataset_id):
         except (errors.HttpError, HttpLib2Error) as e:
             raise api_errors.ApiExecutionError(project_id, e)
 
+    @tracing.traced()
```

Comment:
This decorator is used for classes, use the `@tracing.trace()` instead for class methods or functions

Reply:
Done.